### PR TITLE
Enable login while ldap is offline

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

File: dictionaries/error.definition.json
Comment (longrunningprocess) [inline dictionaries/error.definition.json:28]: Just curious if you have plans for translations yet?  I understand you may still be waiting for approval from @fillup about using Google Translate.  In the meantime, I'll send you a link to the spreadsheet we're using to gather input from our local experts.  I'm thinking you could drop yours in there and Phillip can ping those individuals again.

File: features/login.feature
Comment (longrunningprocess) [inline features/login.feature:181]: cool framework!

**Review by longrunningprocess — APPROVED** [review]

File: dictionaries/error.definition.json
Comment (forevermatt) [reply dictionaries/error.definition.json:28]: I haven't pinged @fillup about that yet. I can certainly add these lines to the spreadsheet, thanks.

**Comment by forevermatt** [comment]
@fillup I'm going ahead and merging this. If you would, please take a look at that last new entry in the [login.feature](https://github.com/silinternational/simplesamlphp-module-silauth/pull/9/files#diff-d392af37bbf2fc7b8e3bd9e1000109e8) file at some point to make sure what it describes is the way we want that scenario to work.

**Pull request merged**
